MiniApp/Classes/MiniApp.swift

```diff
@@ -12,6 +12,17 @@ public class MiniApp: NSObject {
         return RealMiniApp.shared.listMiniApp(completionHandler: completionHandler)
     }
 
+    /// Fetch the MiniAppInfo information for a given MiniApp id.
+    ///
+    /// - Parameters:
+    ///     -   miniAppId: the identifier string of the Mini App you want information
+    ///     -   completionHandler: A block to be called when MiniAppInfo information is fetched. Completion blocks receives the following parameters
+    ///         -   MiniAppInfo: MiniAppInfo information.
+    ///         -   Error: Error details if fetching is failed.
+    public class func info(miniAppId: String, completionHandler: @escaping (Result<MiniAppInfo, Error>) -> Void) {
+        return RealMiniApp.shared.getMiniApp(miniAppId: miniAppId, completionHandler: completionHandler)
```

Comment:
I think it will be good to place validations at the public API for empty inputs of appId. What do you think?

Reply:
you might be right, I'll fix this